src/app/services/core-event-handlers.service.ts

```diff
@@ -21,11 +22,13 @@ export class CoreEventHandlersService {
     private readonly workflow: WorkflowService,
     private readonly localData: LocalDatabaseService,
     private readonly adapterInstall: AdapterInstallService,
-    private readonly translate: TranslateService
+    private readonly translate: TranslateService,
+    private readonly settings: AppSettingsService
   ) {
+    const expose = this.settings.get('exposeCoreActions', false);
     this.formSubmit.actions$
-      .pipe(
-        filter(({ form }) => form === 'core')
+      .pipe(        
+        filter(({ form }) => (form === 'core') || expose)
```

Comment:
I know what this does right now but can it be made explicit with some comments or such? @the-greenman